src/pm/hydra2/mpiexec/mpiexec_params.c

```diff
@@ -847,6 +847,25 @@ static HYD_status tree_width_fn(char *arg, char ***argv)
     goto fn_exit;
 }
 
+/* --pmi-args is an internal option used for singleton mode implementation (MPI-2)
+ * format of option is:
```

Comment:
`--pmi-args` is such a generic name that can be used/implied to all parameters related to `pmi`. If it is only for singleton connection, then use a name that fit, such as `--singleton-args`.